content/dev/docs/timemgr/alphazero.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Time management option alphazero"
+weight: 300
+---
+
+This page contains current and proposed changes to the time manager.
+
+## PR 1201 Time manager alphazero
```

Comment:
No need for PR number in the header actually, it's for users..

.. and now I've noticed it's in dev/

I think it should go to /play/configuration/timemgr/ instead..